docs/core/deploying/deploy-with-cli.md

```diff
@@ -134,7 +134,7 @@ For .NET 2.1, you must use the following switches with the `dotnet publish` comm
   This switch uses an identifier (RID) to specify the target platform. For a list of runtime identifiers, see [Runtime Identifier (RID) catalog](../rid-catalog.md).
 
 - `--self-contained false`
-  This switch tells the .NET Core SDK to create an executable as an FDE.
+  This switch tells the .NET Core SDK to not create a self-contained deployment (SCD), which is the default when -r switch is used. 
```

Comment:
I like this idea. However, I think we should promote FDE here. Mentioning only SCD in the FDE section doesn't seem clear enough.

How does this sound?

```suggestion
  This switch disables the default behavior of the `-r` switch which is to create a self-contained deployment (SCD). Instead, an FDE will be created.
```

Reply:
Cool. Its perfect.

Reply:
Oh sorry -- I'm completely new to pull requests. It was my very first pull request ever.  I thought  I was done after commenting and your suggestion would be applied automatically. Today I checked and it occurred to me that i should commit the suggestion. So I did.  I don't know what else to do.